src/barril/units/_tests/test_fixedarray.py

```diff
@@ -184,3 +184,12 @@ def testFixedArrayIndexAsScalar():
     assert fixed_array.IndexAsScalar(0).GetCategory() == "length of path"
     assert fixed_array.IndexAsScalar(0) == Scalar("length of path", 1, "m")
     assert fixed_array.IndexAsScalar(1) == Scalar("length of path", 2, "m")
+
+
+def testFixedArrayScalarOperations():
+    original = units.FixedArray(3, "length", [100, 80, 50], "m")
```

Comment:
Please test all operations, let's make things proper this time. 😁 

Also test the operation with inverted operands, like you did with `2 - original` below

Reply:
I don't think testing other operations is necessary because they are commutative.
Maybe test exponentiation?

Reply:
I mean, plus, multiply, etc, every implementation that we do implement on `FixedArray`.